### PR TITLE
[ADD] sale_order_type_config: Remove the default from sale order type…

### PR DESCRIPTION
**Pull request by Berezi**
… and purchase operation type.

### CODE REVIEW

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

File: sale_order_type_config/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2021 Berezi - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    type_id = fields.Many2one(default=False, required=True)
```
Comment (oihane) [inline sale_order_type_config/models/sale_order.py]: ERROR dbname odoo.schema: Table 'sale_order': unable to set NOT NULL on column 'type_id'

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA review please.

**Review by Daniel-CA — APPROVED** [review]

**Pull request merged**